StortSpelProjekt/Sandbox/src/main.cpp

```diff
@@ -28,6 +28,8 @@ void DemoUpdateScene(SceneManager* sm);
 
 void DefaultUpdateScene(SceneManager* sm);
 
+EnemyFactory enemyFactory;
```

Comment:
Should we maybe make each scene hold a pointer to their enemy factory? Maybe not in this task though...